EIPS/eip-7303.md

```diff
@@ -0,0 +1,141 @@
+---
+eip: 7303
+title: Token-Controlled Token Circulation
+description: Access control scheme based on token ownership.
+author: Ko Fujimura (@kofujimura)
+discussions-to: https://ethereum-magicians.org/t/erc-7303-token-controlled-token-circulation/15020
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-09
+requires: 721
```

Comment:
```suggestion
requires: 721, 5679
```
Since 5679 is being specified in the specification, it will needs to be mentioned in the "requires" field

Reply:
Since the requirement level of ERC-5679 is "SHOULD" in this specification, I did not specify it in this field. But, your suggest is that this is needed even "SHOULD" level, am I right? If so, I will add it.

Reply:
I added 1155 and 5679 in the new version. Thank you.